services/query.proto

```diff
@@ -197,5 +198,10 @@ message Query {
          * Gets all information about an account including allowances granted by the account
          */
         GetAccountDetailsQuery accountDetails = 58;
+
+        /**
+         * Get all information about a node
+         */
+        NodeGetInfoQuery nodeGetInfo = 59;
```

Comment:
Found post merge, this doesn't compile (because we used bad package names in these legacy files)...
fix:
```suggestion
        com.hedera.hapi.node.addressbook.NodeGetInfoQuery nodeGetInfo = 59;
```

Reply:
I have another PR in hedera-service to compile all the protobuf changes. This change matches others. We do have import "node_get_info.proto"; at the top.

Reply:
It fails to compile as a protobuf.
No idea why your services PR works (might be a bug in PBJ), but it doesn't compile in protoc.

Reply:
@jsync-swirlds that's bizarre... 🤯

Reply:
You can see this [PR](https://github.com/hashgraph/hedera-services/pull/13090) is compiled. Query.java and NodeGetInfoQuery.java were generated correctly.

Reply:
PBJ doesn't really used a lot of the directives in the proto file as it only handles Java, and there is much simplification (which sometimes leads to bugs or overly-relaxed compile as it does here).
protoc is strict about all of the directives, including the `package` keyword (which it treats as a C++ namespace for the messages, which has interesting semantics).
protoc is used for everything except the consensus node software, so we have to continue working with protoc.

Reply:
Is this a new guideline for services?

Reply:
We have always required protoc, it's what builds all our SDKs.
It's just not always been well communicated.

Reply:
Disregard.

There were a couple cross-merged branches that caused a false error.
The issue has been fixed.

Reply:
To item (3):
Services should not run protoc.
This protobuf repo should, however.  That's the more critical missing piece, there are no CI checks in this repo, so it's far too easy to accidentally commit a change here that doesn't compile everywhere it needs to (i.e. Javascript, Go, C#, C++, Rust, Java, etc...).

When you build in services, it runs PBJ, no protoc.  Sometimes PBJ will succeed when protoc will not, and it only generates the java sources (not all the other languages).  Again, not a services issue, but an issue to resolve in _this_ repo.